MAUI/Rating/Overview.md

```diff
@@ -0,0 +1,24 @@
+---
+
+layout: post
+title: About .NET MAUI Rating control | Syncfusion
+description: Learn here all about the introduction of Syncfusion .NET MAUI Rating (SfRating) control, its elements and more.
+platform: maui
+control: Rating
+documentation: ug
+
+---
+# .NET MAUI Rating (SfRating) Overview
+
+The Essential .NET MAUI Rating control provides the number of stars that represents a rating. And also used to configure the item size, item spacing and the number of displayed items in the [`SfRating`] control. Essential .NET MAUI Rating control can be used in various scenarios like movie rating, rating the application etc.
```

Comment:
The Essential .NET MAUI Rating control provides the number of stars that represents a rating. Also used to configure the item size, item spacing, and the number of displayed items in the [`SfRating`] control. Essential .NET MAUI Rating control can be used in various scenarios like movie rating, rating the application, etc.

Reply:
Changed